index.js

```diff
@@ -1,6 +1,78 @@
 const express = require('express');
+const jsonfile = require('jsonfile');
 
-// const jsonfile = require('jsonfile');
+//getting the data from the json file
+var file = './pokedex.json'
+var object;
+jsonfile.readFile(file, function(err, obj) {
+  object = obj;
+})
```

Comment:
Nice work! 

Due to the asynchronousity, I'll recommend putting this line right at the bottom.
So that obj can be accessible thru out your code.

Reply:
just line 9 only.